scrapy_zyte_api/_params.py

```diff
@@ -629,32 +726,21 @@ def parse(self, request):
             cookies_enabled=cookies_enabled,
             cookie_jars=self._cookie_jars,
             max_cookies=self._max_cookies,
+            experimental_cookies=self._experimental_cookies,
         )
-        if not dont_merge_cookies and self._warn_on_cookies:
-            self._handle_warn_on_cookies(request, params)
+        if (
+            params
+            and self._unreported_deprecated_experimental_fields
+            and "experimental" in params
+        ):
+            for field in list(self._unreported_deprecated_experimental_fields):
+                if field in params["experimental"]:
+                    self._unreported_deprecated_experimental_fields.remove(field)
```

Comment:
Went with this to warn only once. I wonder if it is overkill, or even if it is better to log a warning on every request to encourage a fix and minimize the chance of users missing the warning message (i.e. if you are already ignoring a couple of warnings, you might not notice the extra one).